test/nested-frames/target.js

```diff
@@ -0,0 +1,59 @@
+/**
```

Comment:
It's easier to have them all in one place rather than create a new test for every case.